README.md

```diff
@@ -30,25 +54,225 @@ npm install --save ipfsd-ctl
 
 IPFS daemons are already easy to start and stop, but this module is here to do it from JavaScript itself.
 
+### Local node
+
 ```js
 // Start a disposable node, and get access to the api
-// print the node id, and kill the temporary daemon
+// print the node id, and stop the temporary daemon
 
 // IPFS_PATH will point to /tmp/ipfs_***** and will be
 // cleaned up when the process exits.
 
-var ipfsd = require('ipfsd-ctl')
+const daemonFactory = require('ipfsd-ctl')
+const local = daemonFactory.localController
 
-ipfsd.disposableApi(function (err, ipfs) {
-  ipfs.id(function (err, id) {
+local.spawn(function (err, ipfsd) {
+  const ipfsCtl = ipfsd.ctl
+  const ipfsCtrl = ipfsd.ctrl
+  ipfsCtl.id(function (err, id) {
     console.log(id)
-    process.exit()
+    ipfsCtrl.stopDaemon()
   })
 })
 ```
 
-If you need want to use an existing ipfs installation you can set `$IPFS_EXEC=/path/to/ipfs` to ensure it uses that.
+### Remote node
+
+```js
+// Start a remote disposable node, and get access to the api
+// print the node id, and stop the temporary daemon
+
+// IPFS_PATH will point to /tmp/ipfs_***** and will be
+// cleaned up when the process exits.
+
+const daemonFactory = require('ipfsd-ctl')
+const server = daemonFactory.server
+
+server.start((err) => {
```

Comment:
What are the defaults? Better add here an options object and exemplify that config.